api/src/test/java/io/opentelemetry/metrics/DoubleCounterTest.java

```diff
@@ -87,6 +88,8 @@ public void noopBind_WithBadLabelSet() {
   public void doesNotThrow() {
```

Comment:
we should have a test to verify you can't pass in a negative increment

Reply:
Done.